aws_helper.sh

```diff
@@ -112,6 +116,39 @@ EOF
   unset AWS_ARN;
 }
 
+##
+# Get list of credentials from users config files
+##
+function __aws_helper_list_credentials() {
+ if [ "${1}" == "help" ]; then
+      cat <<EOF
+List AWS environment credentials in user configuration
+
+Usage: aws-helper list-creds [OPTIONS]
+
+Options:
+  --file <credentials filename>
+
+  default = ~/.aws/credentials
+EOF
+      return 0;
+  fi
+
+  ## Do we have the tools
+
+  local GREP=$(which ggrep) || local GREP=${which grep} || __aws_helper_log 'error' 'Cannot locate tool: grep';
```

Comment:
This line does not work on my local machine with GNU Bash 4.4.2. This could be due to the fact that the first `which` call fails, and the second isn't executed. You may have to change this to check the exit codes of each subshell.

Also I believe `${which grep}` needs to be `$(which grep)`